src/shared/components/login/login.css

```diff
@@ -35,6 +35,13 @@ select {
   flex-direction: column;
 }
 
+.flexRow {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
```

Comment:
Change `justify-content: space-between;` to `justify-content: center;`

Reply:
done.